### ISSUE TITLE
CelebA for training

### ISSUE THREAD

**Issue by thinkerthinker** [Prinsphield/ELEGANT#15]
Are you using all the pictures of CelebA for training? I didn't find the test set and training set in your code.

**Comment by Prinsphield** [comment]
Yes. You can split the training data and test data by yourself if you'd love to.
